### DISCUSSION TITLE
V0.7.6(Web1.5.4.1)

### DISCUSSION THREAD

**Discussion by cloud-fs** [cloud-fs/cloud-fs.github.io#100] [Announcements]
V0.7.6(Web1.5.4.1)

1. 修复0.7.4版本新用户添加网盘失败的问题

<hr /><em>This discussion was created from the release <a href='https://github.com/cloud-fs/cloud-fs.github.io/releases/tag/v0.7.6'>V0.7.6(Web1.5.4.1)</a>.</em>

**Comment by truexeen** [comment]
请问安装在windows 10下，访问webdav功能出现http 405错误，有什么解决办法吗？
